deps/build.jl

```diff
@@ -37,6 +37,10 @@ println("- extra flags = $extra")
 
 run(`$(clang()) $files $CXXFLAGS $LDFLAGS $extra $libs $output`)
 
+# cleaning up
+println("Cleaning up...")
+rm.(filter(contains(r".jl$"), readdir(joinpath(@__DIR__, "..", "src", "dialects"); join=true)))
```

Comment:
Why not use endswith here?
```suggestion
rm.(filter(endswith(".jl"), readdir(joinpath(@__DIR__, "..", "src", "dialects"); join=true)))
```